### ISSUE TITLE
Glasklart ShareSheet Theme

### ISSUE THREAD

**Issue by dreamnet** [glasklart/hd#7722]
Now that [Anemone](https://github.com/AnemoneTeam/Anemone) has the ability to [theme the ShareSheet icons](https://github.com/AnemoneTeam/Anemone/wiki/Share-Sheet-Icons) I have made a Glasklart theme for the Share sheet (see commit below).
All icons made 180x180px - so this should work on the 6+ too.

**Comment by dreamnet** [comment]
All seem to be fine - Closing.

**Comment by dreamnet** [comment]
There are two missing icons:

- Gift (App Store) and
- Add to Wish List (App Store)

I'm currently working on them.

**Comment by grkka** [comment]
![unnamed](https://cloud.githubusercontent.com/assets/14794477/17978751/c981adc6-6af7-11e6-88d8-d95f43743e25.png)

**Comment by dreamnet** [comment]
@grkka Yes, that's what I mean. I have all icons ready - they will be in release 4.40.

**Comment by ivanov007** [comment]
ok, so also Copy Link and More, also for me there's clips and firefox... I'll try to create those

**Comment by dreamnet** [comment]
[Here](https://github.com/glasklart/hd/tree/master/Library/Themes/Glasklart%20ShareSheet.theme/IconBundles) are all Share Sheet icons (except `Gift` and `Add to Whish List`, I have them ready, but can't commit them to the master at the moment, because I am working on an other commit).
**There should be no need for more Share Sheet icons.**

**Comment by dreamnet** [comment]
`Gift` and `Add to Wish List` are now also themed.

**Comment by grkka** [comment]
![asset](https://cloud.githubusercontent.com/assets/14794477/18002943/3aea0dc6-6b8b-11e6-81b1-f3dcd7536e17.PNG)
???

**Comment by grkka** [comment]
![asset](https://cloud.githubusercontent.com/assets/14794477/18170820/ff90f8be-705f-11e6-898d-2587a4cb469b.PNG)

**Comment by Trubbel** [comment]
@grkka Strange, it is themed for me.

![file 01-09-16 16 48 09](https://cloud.githubusercontent.com/assets/10730122/18171943/fd773e18-7063-11e6-9fb0-6c158b379425.png)

**Comment by ivanov007** [comment]
yeah it's working for me too, @grkka can you check if you have sharesheet enabled on anemone?

**Comment by grkka** [comment]
![asset](https://cloud.githubusercontent.com/assets/14794477/18175396/730f10c6-7071-11e6-820b-e0cb6d762bfd.PNG)
![asset 1](https://cloud.githubusercontent.com/assets/14794477/18175401/75e66eb6-7071-11e6-84dc-604ed5130b41.PNG)
![asset 2](https://cloud.githubusercontent.com/assets/14794477/18175406/781eb74c-7071-11e6-8844-3514a800ee41.PNG)
I unchecked and have again set the option to sharesheet, I have applied the changes, and now Filza if it looks good. But Tweetbot still appear different icons. @ivanov007 @Trubbel

**Comment by Checkit23** [comment]
In TweetBot, there's a single icon that's not themed by Glasklart yet. The icon can be found in the ShareSheet inside the embedded browser.

![tweetbot sharesheet](https://cloud.githubusercontent.com/assets/16171251/18257153/abceb154-73bf-11e6-80cd-8e3266d4ca59.jpg)

**Comment by dreamnet** [comment]
@Checkit23 Please report this to @coolstar [here](https://github.com/AnemoneTeam/Anemone/issues/159). Anemone is currently not able to theme this icon.

**Comment by coolstar** [comment]
hmm, PTHTweetbotCopyLinkUIActivity-large.png doesn't work?

**Comment by dreamnet** [comment]
@coolstar PTHTweetbotCopyLinkUIActivity-large.png is [this icon](https://github.com/glasklart/hd/blob/master/Library/Themes/Glasklart%20ShareSheet.theme/IconBundles/PTHTweetbotCopyTweetUIActivity-large.png).

**Comment by coolstar** [comment]
interesting. I'll take a look in a few

**Comment by dreamnet** [comment]
Here are all icons from Tweetbot's `Assets.car`: [Tweetbot_Assets.zip](https://github.com/glasklart/hd/files/455645/Tweetbot_Assets.zip)
The image that can't be themed is `activity_tweet.png` - but there are more `activity_...` images...

**Comment by 2Qt2beStr8** [comment]
@dreamnet  i think we need some more ShareSheet Icons :-)

**Comment by dreamnet** [comment]
@2Qt2beStr8 No, normally not - all icons in upper line (except "More") should be themed with there normal app icons (Glasklart Icons.theme/IconBundles/). Do you have this activated in Anemone? What app you are using here?
Here is all I know: https://github.com/AnemoneTeam/Anemone/wiki/Share-Sheet-Icons

**Comment by 2Qt2beStr8** [comment]
Yes, i have ShareSheet activated

![f8bc1272-afa7-4c9e-afad-232e019ef16d](https://user-images.githubusercontent.com/20930745/36626594-c4881718-1935-11e8-9c6b-b6d6fded90da.jpeg)

**Comment by kayjeiem** [comment]
Does this work on iOS 13.3, Unc0ver 4.3.1? I've tried using iThemer and Snowboard but nothing.

**Comment by dreamnet** [comment]
@kayjeiem Seems that SnowBoard dosent support theming the share sheet. 😢
